fitbenchmarking/utils/tests/test_options.py

```diff
@@ -43,7 +43,8 @@ def setUp(self):
                             'num_runs': 2,
                             'software': ['foo', 'bar']},
                 'PLOTTING': {'colour_scale': [(17.1, 'a_string?'),
-                                              (float('inf'), 'another_string')],
+                                              (float('inf'),
+                                              'another_string')],
```

Comment:
Should this line up with the open bracket for the tuple?
```suggestion
                                               'another_string')],
```

Reply:
Done! Nice catch